spec/Appendix A -- Persisted Documents.md

```diff
@@ -95,6 +95,50 @@ implementation specific.
 Note: A 32 character hexadecimal _custom document identifier_ is likely to be an
 MD5 hash of the GraphQL document, as traditionally used by Relay.
 
+### Document identifier syntax
+
+DocumentIdentifier ::
+
+- PrefixedDocumentIdentifier
+- CustomDocumentIdentifier
+
+PrefixedDocumentIdentifier ::
+
+- Sha256HexDocumentIdentifier
+- OtherPrefixedDocumentIdentifier
+
+Sha256HexDocumentIdentifier :: `sha256:` LowerCaseHexDigit+
+
+LowerCaseHexDigit :: one of
+
+- `0` `1` `2` `3` `4` `5` `6` `7` `8` `9`
+- `a` `b` `c` `d` `e` `f`
+
+OtherPrefixedDocumentIdentifier :: `x-` UnreservedCharacter+ `:`
+UnreservedCharacter+
```

Comment:
Pursuant to the 7-25-2024 discussion, to allow additional `:` characters within the payload, this definition needs to be updated as it currently only allows unreserved characters in both the prefix and payload.

Reply:
Addressed [here](https://github.com/graphql/graphql-over-http/pull/296/files#diff-9be5577e05ae2112d2b8f95584b162d0dec01453bf6c85df58bf5db4f2c9727aR121)

Reply:
I like it!